control_nodes:
- {row: 1, col: 1, x: 0.17, y: -0.679, z: 0.571, qx: 1.0, qy: -0.024, qz: 0.011, qw: -0.014}
- {row: 1, col: 2, x: 0.17, y: -0.679, z: 0.571, qx: 1.0, qy: -0.024, qz: 0.011, qw: -0.014}
- {row: 1, col: 4, x: 0.177, y: -0.686, z: 0.57, qx: 1.0, qy: -0.024, qz: 0.011, qw: -0.014}
- {row: 1, col: 6, x: 0.183, y: -0.691, z: 0.57, qx: 1.0, qy: -0.024, qz: 0.011, qw: -0.014}
- {row: 1, col: 8, x: 0.189, y: -0.696, z: 0.571, qx: 1.0, qy: -0.024, qz: 0.011,
  qw: -0.014}
- {row: 1, col: 10, x: 0.195, y: -0.702, z: 0.571, qx: 1.0, qy: -0.024, qz: 0.011,
  qw: -0.014}
- {row: 1, col: 12, x: 0.201, y: -0.708, z: 0.571, qx: 1.0, qy: -0.024, qz: 0.011,
  qw: -0.014}
- {row: 2, col: 2, x: 0.165, y: -0.685, z: 0.567, qx: 1.0, qy: -0.024, qz: 0.009,
  qw: -0.016}
- {row: 2, col: 4, x: 0.17, y: -0.693, z: 0.57, qx: 1.0, qy: -0.024, qz: 0.011, qw: -0.014}
- {row: 2, col: 6, x: 0.176, y: -0.697, z: 0.57, qx: 1.0, qy: -0.024, qz: 0.011, qw: -0.014}
- {row: 2, col: 8, x: 0.182, y: -0.7025, z: 0.57, qx: 1.0, qy: -0.024, qz: 0.011, qw: -0.014}
- {row: 2, col: 10, x: 0.188, y: -0.708, z: 0.57, qx: 1.0, qy: -0.024, qz: 0.011,
  qw: -0.014}
- {row: 2, col: 12, x: 0.195, y: -0.715, z: 0.571, qx: 1.0, qy: -0.024, qz: 0.011,
  qw: -0.014}
control_cols: [1, 4, 8, 12, 16]
grid: {rows: 2, cols: 10}
